src/App.tsx

```diff
@@ -10,33 +12,29 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
+  const [selectedTabId, setSelectedTabId] = useState('tab-1');
+
+  const currentTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  const onTabSelected = (tab: Tab) => {
```

Comment:
```suggestion
  const handleTabSelected = (tab: Tab) => {
```
https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364